styles.css

```diff
@@ -1 +1,10 @@
-
+#img-div {
+    width: 100%;
+    text-align: center;
+  }
+  
+  #image {
```

Comment:
Please, a `class` will be more flexible than an `id`

`.image`

You only use `ids` for very specific cases 😃